packages/engine/paima-sm/src/types.ts

```diff
@@ -145,6 +161,7 @@ export enum CdeEntryTypeName {
   ERC721 = 'erc721',
   ERC6551Registry = 'erc6551-registry',
   CardanoDelegation = 'cardano-stake-delegation',
+  CardanoDelayedNft = 'cardano-delayed-nft',
```

Comment:
I think we have to change this name. I don't really mind if it's one CDE or two CDEs (if we change our minds later, we can just change the code). It should be something more general than `nft` like "token" or "asset" though